source/MapPanel.cpp

```diff
@@ -440,16 +447,24 @@ bool MapPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, bool
 		GetUI()->Pop(this);
 		GetUI()->Push(new MapDetailPanel(*this));
 	}
+	else if(key == 'y' && isStarry == false)
+	{
+		return isStarry = true;
+	}
+	else if(key == 'y' && isStarry == true)
+	{
+		return isStarry = false;
+	}
```

Comment:
Change these lines (450-457)with a simple inverter;
```cpp
else if(key == "y")
    isStary = !isStary;
```
Why do you return the state of isStary? I cant see the bigger picture but it seems like We return true if a key was pressed?